cmd/init.go

```diff
@@ -68,14 +68,15 @@ var InitCmd = &cobra.Command{
 }
 
 func init() {
-	InitCmd.Flags().BoolVarP(&kubernetesMode, "kubernetes", "k", false, "Deploy Dapr to a Kubernetes cluster")
-	InitCmd.Flags().BoolVarP(&slimMode, "slim", "s", false, "Initialize dapr in self-hosted mode without placement, redis and zipkin containers.")
-	InitCmd.Flags().StringVarP(&runtimeVersion, "runtime-version", "", "latest", "The version of the Dapr runtime to install. for example: v0.1.0")
+	InitCmd.Flags().BoolVarP(&kubernetesMode, "kubernetes", "k", false, "Deploys Dapr to a Kubernetes cluster")
+	InitCmd.Flags().BoolVarP(&slimMode, "slim", "s", false, "Excludes placement service, Redis and Zipkin containers from self-hosted installation")
+	InitCmd.Flags().StringVarP(&runtimeVersion, "runtime-version", "", "latest", "The version of the Dapr runtime to install, for example: 1.0.0")
 	InitCmd.Flags().StringVarP(&initNamespace, "namespace", "n", "dapr-system", "The Kubernetes namespace to install Dapr in")
-	InitCmd.Flags().BoolVarP(&enableMTLS, "enable-mtls", "", true, "Enable mTLS in your cluster. Default: true")
-	InitCmd.Flags().BoolVarP(&enableHA, "enable-ha", "", false, "Deploy Dapr in a highly available mode. Default: false")
+	InitCmd.Flags().BoolVarP(&enableMTLS, "enable-mtls", "", true, "Enables mTLS in your cluster")
```

Comment:
You seem to have removed all the default descriptions. Is that intentional? I would keep these. For example
Should be "Enable mTLS in your cluster (default true)"

Also this should be "enable" not "enables"

Reply:
It is intentional because the library handles that automatically based on the provided default value.

Reply:
We are also making description in the 3rd person. <Command> <verb> ...